### ISSUE TITLE
frames and ms in "Duration" do not correlate

### ISSUE THREAD

**Issue by ichbinmatthias** [wonderunit/storyboarder#1076]
When changing the duration for a board the frame number always switches to 24 after a while, while the duration in ms shows the correct number. 

operating system is Win 10

**Comment by audionerd** [comment]
Is this when using the Timeline view to drag the duration?

**Comment by ichbinmatthias** [comment]
no, I type in the frame number in the field right of the canvas under "Duration". I had no idea you can drag the duration of a panel in the timeline? Even now I cannot figure out how to do that...

Maybe I should mention that I currently only have that problem in one project. Just started a new storyboard and the issue doesn't occur there.

**Comment by Woutersel** [comment]
Same here. Duration always reverts to 50 frames when typing in ms or frames.
The numbers of the ms are correct, but frames is always 50.
It's also problematic with Duduf's script "WUSToryboarderAE": I can't load the .storyboarder project in AE.

**Comment by mMrJay** [comment]
This happens to me too, even though I changed the duration of the board it still shows the default in frames.
